free-2.md

```diff
@@ -58,12 +105,25 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+A lot of the time would be spent on figuring out good ways to interact with the
+user. Keeping track of events is pretty easy and notifications can be anywhere
+from easy to pretty difficult. However, the user interaction can also be pretty
+incremental: adding support for manual input would be one level, gmail input
+would be harder, and other methods of input and output could get trickier.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is potentially a big project depending on how many different devices and
+platforms it supports, but could pretty easily be toned down to something
+reasonable. If it does get big and is done well, it could be very useful and I
+could definitely see myself using it.
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+This is a good idea for a project because it is very language-y and useful.
+There are some drawbacks in that supporting enough different methods of input
+and output could be technically difficult, but that is all part of the language
+design.
```

Comment:
ATM the current description doesn't sound very language-y to me at all. But, lots of *design*.

You also wouldn't want to get hung up on interface beauty, because that alone is an infinite task.